lib/libutils/ext/include/util.h

```diff
@@ -55,6 +55,21 @@
 		(void)(*(res) = __roundup_tmp & ~__roundup_mask), 0; \
 }))
 
+/*
+ * Rounds up the even multiple of y and then divides by y.  Safe against
+ * overflow, y has to be a multiple of 2.
```

Comment:
Missing "to", and I'm confused by the use of "even" (in the above comment too). It could be understood as `(n * y)` with `n` being even, which is not the case. Only the result is even, but that should be obvious since `y` has to be even to begin with. So I suggest:  "Rounds up to the nearest multiple of y and then divides by y".

Reply:
Much better, thank you.